Drop-In/src/main/java/com/braintreepayments/api/PaymentMethodInspector.java

```diff
@@ -0,0 +1,112 @@
+package com.braintreepayments.api;
+
+import com.braintreepayments.cardform.utils.CardType;
+
+class PaymentMethodInspector {
```

Comment:
Curious the reasoning here for making this it's own file vs including within `DropInPaymentMethod`?

Reply:
Yeah this sort of became a junk drawer for code that may be more suitable for the core sdk. Another goal was to make `DropInPaymentMethod` more of a pure value type, although it may make sense in `v5` to create a `PaymentMethod` enum in core instead.